script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2.txt

```diff
@@ -28,7 +28,7 @@ My head is numb from all the unbelievable things that have happened since this m
 This is the second time I've said that today.
 The first time was directed towards Arcueid, but this time it's directed at myself.
 I can't concentrate because I can hear the sound of the sleeping woman's breathing.
-Even for someone as unprincipled as me, there's got to be a limit. Why am I getting so excited about her now, of all times...!
+Even for someone as unprincipled as me, there's got to be a limit. Why am I getting so excited about her now, of all times...?!
```

Comment:
Kinda prefer this one as just a !, he's not _really_ asking a question here

Reply:
Slight preference for the single ! too